src/game.cpp

```diff
@@ -4297,7 +4297,7 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats,
 inline static const char *yawToDirectionString(int yaw)
 {
 	const char *player_direction;
-	std::string direction[4] = {"North [+Z]", "West [-X]", "South [-Z]", "East [+X]"}
+	const char *direction[4] = {"North [+Z]", "West [-X]", "South [-Z]", "East [+X]"};
```

Comment:
It needs to be `static const char *direction[4] = {"North [+Z]", "West [-X]", "South [-Z]", "East [+X]"};`